ui/decredmaterial/editor.go

```diff
@@ -173,27 +147,78 @@ func (e Editor) Layout(gtx layout.Context) layout.Dimensions {
 	})
 }
 
-func (e Editor) editorLine(gtx C) D {
+func (e Editor) editorLayout(gtx C) D {
+	var dims layout.Dimensions
+	switch e.LineStyle {
+	case RoundedRectangle:
+		dims = e.editorRectangle(gtx, func(gtx C) D {
+			return e.editorSection(gtx, false)
+		})
+	case SingleUnderLine:
+		dims = e.editorSection(gtx, true)
+	case NoLine:
+		dims = e.editorSection(gtx, false)
```

Comment:
Do we really need to have 3 editor layouts? The entire app would be using the editor with rounded borders. I'm not sure where we might need a bare editor with no border but a simple if statement with a property in the Editor struct should be able to resolve that.

Reply:
The bare editor is what the send page amount editor uses.